### DISCUSSION TITLE
如何更改搜索關鍵字

### DISCUSSION THREAD

**Discussion by u09f19** [1715173329/UnblockNeteaseMusic#1386] [Q&A]
目前使用yt-dlp這個音源在聽日文歌，但是遇到歌手跟歌曲名稱都是英文的時候容易找錯
自己試著搜尋了幾首目前找錯的歌曲，發現在後面加入japan可以找到正確的歌曲，想改關鍵字搜索試試

但是在yt-dlp.js裡面把這行後面加入japan後並沒有任何變化
const byKeyword = (keyword) => 'ytsearch1:${keyword} japan';

想請問要怎麼修改

**Comment by pan93412** [comment]
🤔 現在還有這個問題嗎？可能要加引號吧 `ytsearch1:"${keyword} japan"`

**Reply by u09f19** [reply]
當初試了一陣子都沒成功就沒繼續試了
加了引號還是不行😢

有測試過如果直接使用yt-dlp.js的話可以更改關鍵字
但是運行整個程式的時候就沒任何改變了，不知道是不是還要改別的地方
